src/main/java/com/adyen/model/AbstractPaymentRequest.java

```diff
@@ -758,6 +756,9 @@ public ApplicationInfo getApplicationInfo() {
     }
 
     public void setApplicationInfo(ApplicationInfo applicationInfo) {
+        if (applicationInfo == null) {
+            throw new IllegalArgumentException("ApplicationInfo cannot be null");
+        }
```

Comment:
We should not throw exception here.

Reply:
We decided to remove the setter, so this is not an issue anymore.